openmetadata-ui/src/main/resources/ui/src/constants/mockTourData.constants.ts

```diff
@@ -3454,30 +3460,36 @@ export const mockSearchData = {
           {
             _index: 'table_search_index',
             _type: '_doc',
-            _id: 'c0057714-2d87-4bb9-a300-7032676fc6de',
-            _score: 5,
+            _id: '644405ad-60fe-4329-884f-fc3e3ab6c6a5',
+            _score: 5.028171,
```

Comment:
Do we need this score to have decimal precision?

Reply:
Hey just took updated response from api.